gammapy/modeling/models/__init__.py

```diff
@@ -1,13 +1,126 @@
+    ConstantFluxSpatialModel,
+    TemplateSpatialModel,
+)
+from .spectral import (
+    scale_plot_flux,
+    integrate_spectrum,
+    SpectralModel,
+    ConstantSpectralModel,
+    CompoundSpectralModel,
+    PowerLawSpectralModel,
+    PowerLawNormSpectralModel,
+    PowerLaw2SpectralModel,
+    BrokenPowerLawSpectralModel,
+    SmoothBrokenPowerLawSpectralModel,
+    PiecewiseNormSpectralModel,
+    ExpCutoffPowerLawSpectralModel,
+    ExpCutoffPowerLawNormSpectralModel,
+    ExpCutoffPowerLaw3FGLSpectralModel,
+    SuperExpCutoffPowerLaw3FGLSpectralModel,
+    SuperExpCutoffPowerLaw4FGLSpectralModel,
+    LogParabolaSpectralModel,
+    LogParabolaNormSpectralModel,
+    TemplateSpectralModel,
+    ScaleSpectralModel,
+    EBLAbsorptionNormSpectralModel,
+    NaimaSpectralModel,
+    GaussianSpectralModel,
+)
+
+from .spectral_cosmic_ray import create_cosmic_ray_spectral_model
+from .spectral_crab import (
+    MeyerCrabSpectralModel,
+    create_crab_spectral_model,
+)
+from .temporal import (
+    TemporalModel,
+    ConstantTemporalModel,
+    LinearTemporalModel,
+    ExpDecayTemporalModel,
+    GaussianTemporalModel,
+    LightCurveTemplateTemporalModel,
+    PowerLawTemporalModel,
+    SineTemporalModel,
+)
+
+
+
+__all__ = [
+    "create_fermi_isotropic_diffuse_model",
+    "DatasetModels",
+    "FoVBackgroundModel",
+    "Model",
+    "MODEL_REGISTRY",
+    "ModelBase",
+    "Models",
+    "SPATIAL_MODEL_REGISTRY",
+    "SPECTRAL_MODEL_REGISTRY",
+    "TemplateNPredModel",
+    "TEMPORAL_MODEL_REGISTRY",
+    'BrokenPowerLawSpectralModel',
+    'CompoundSpectralModel',
+    'ConstantFluxSpatialModel',
+    'ConstantSpatialModel',
+    'ConstantSpectralModel',
+    'ConstantTemporalModel',
+    'create_cosmic_ray_spectral_model',
+    'create_crab_spectral_model',
+    'DiskSpatialModel',
+    'EBLAbsorptionNormSpectralModel',
+    'ExpCutoffPowerLaw3FGLSpectralModel',
+    'ExpCutoffPowerLawNormSpectralModel',
+    'ExpCutoffPowerLawSpectralModel',
+    'ExpDecayTemporalModel',
+    'GaussianSpatialModel',
+    'GaussianSpectralModel',
+    'GaussianTemporalModel',
+    'GeneralizedGaussianSpatialModel',
+    'integrate_spectrum',
+    'LightCurveTemplateTemporalModel',
+    'LinearTemporalModel',
+    'LogParabolaNormSpectralModel',
+    'LogParabolaSpectralModel',
+    'MeyerCrabSpectralModel',
+    'NaimaSpectralModel',
+    'PiecewiseNormSpectralModel',
+    'PointSpatialModel',
+    'PowerLaw2SpectralModel',
+    'PowerLawNormSpectralModel',
+    'PowerLawSpectralModel',
+    'PowerLawTemporalModel',
+    'scale_plot_flux',
+    'ScaleSpectralModel',
+    'Shell2SpatialModel',
+    'ShellSpatialModel',
+    'SineTemporalModel',
+    'SkyModel',
+    'SmoothBrokenPowerLawSpectralModel',
+    'SpatialModel',
+    'SpectralModel',
+    'SuperExpCutoffPowerLaw3FGLSpectralModel',
+    'SuperExpCutoffPowerLaw4FGLSpectralModel',
+    'TemplateSpatialModel',
+    'TemplateSpectralModel',
+    'TemporalModel',
+]
```

Comment:
please, use double quotes as in proposed by [black](https://github.com/psf/black) formatting style